wiki/Community/Project_Loved/en.md

```diff
@@ -49,7 +47,8 @@ Every month, beatmaps are chosen according to 9 categories, with the goal of ens
 
 | Category | Explanation |
 | :-: | :-- |
-| Popular Playcounts | Popular beatmaps in the community, with at least 100,000 playcount. |
+| Popular Playcounts 1 | Popular beatmaps in the community, with at least 100,000 playcount. |
+| Popular Playcounts 2 | Popular beatmaps in the community, with at least 100,000 playcount. |
```

Comment:
mentioning this twice feels redundant imo.
```suggestion
```

Reply:
Woudln't 2 Popular Playcounts be better to imply that it's 2 maps for that category.
Popular Playcount 2 sounds like it describes a different ruleset as Popular Playcount 1 (it's obviously the same but it sounds that way)